What's new article (#271)
dotnet/docs-aspire: docs/whats-new/dotnet-docs-aspire-mod0.md

---
title: ".NET Aspire docs: What's new for December 2023"
description: "What's new in the .NET Aspire docs for December 2023."
ms.custom: December-2023
ms.date: 01/01/2024
---

# .NET Aspire docs: What's new for December 2023

Welcome to what's new in the .NET Aspire docs for December 2023. This article lists some of the major changes to docs during this period.

> ## Storage

Reviewer: ```suggestion
## Deployment

### New articles

- [Deploy a .NET Aspire app to Azure Container Apps using the Azure Developer CLI (in-depth guide)](../deployment/azure/aca-deployment-azd-in-depth.md)

### Updated articles

- [Deploy a .NET Aspire app to Azure Container Apps using the Azure Developer CLI (in-depth guide)](../deployment/azure/aca-deployment-azd-in-depth.md)
  - Aca deployment tweaks
  - ACA deployment with AZD

## Storage
```

### New articles

- [Title not found in: #108](../storage/azure-storage.md)

## Database

### New articles

- [.NET Aspire MongoDB database component](../database/mongodb-component.md)
- [.NET Aspire MySQL database component](../database/mysql-component.md)
- [Title not found in: #131](../database/quickstart-sql-server.md)

## Deployment

### New articles

- [Title not found in: #134](../deployment/k8s-deployment.md)

### Updated articles

> - [.NET Aspire manifest format for deployment tool builders](../deployment/manifest-format.md) - Manifest and resources

Reviewer: ```suggestion
- [.NET Aspire manifest format for deployment tool builders](../deployment/manifest-format.md) - Manifest and resources
- [Deploy a .NET Aspire app to Azure Container Apps using the Azure Developer CLI (in-depth guide)](../deployment/azure/aca-deployment-azd-in-depth.md)
	  - Aca deployment tweaks
	  - ACA deployment with AZD
```

## Community contributors

The following people contributed to the .NET Aspire docs during this period. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).

> - [alexwiese](https://github.com/alexwiese) - Alex Wiese ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
> - [ericmutta](https://github.com/ericmutta) - Eric Mutta ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
> - [peteraritchie](https://github.com/peteraritchie) - Peter Ritchie ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
> - [smargoli2](https://github.com/smargoli2) - Sora Teichman ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)

Reviewer: ```suggestion
- [peteraritchie](https://github.com/peteraritchie) - Peter Ritchie ![4 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-4-green)
- [alexwiese](https://github.com/alexwiese) - Alex Wiese ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
- [ericmutta](https://github.com/ericmutta) - Eric Mutta ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
- [georgiosd](https://github.com/georgiosd) - Georgios Diamantopoulos ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
- [HTSchoenfelder](https://github.com/HTSchoenfelder) - Henrik Schönfelder ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
- [jasond-s](https://github.com/jasond-s) - Jason Dryhurst-Smith ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
- [Roman-Blinkov](https://github.com/Roman-Blinkov) - Roman Blinkov ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
- [smargoli2](https://github.com/smargoli2) - Sora Teichman ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
- [ZieMcd](https://github.com/ZieMcd) - Zie Mcdowell ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
```

---
title: ".NET Aspire docs: What's new for December 2023"
description: "What's new in the .NET Aspire docs for December 2023."
ms.custom: December-2023
ms.date: 01/01/2024
---

# .NET Aspire docs: What's new for December 2023

Welcome to what's new in the .NET Aspire docs for December 2023. This article lists some of the major changes to docs during this period.

## Storage

### New articles

> - [Title not found in: #108](../storage/azure-storage.md)

Reviewer: ```suggestion
- [Tutorial: Connect an ASP.NET Core app to .NET Aspire storage components](../storage/azure-storage.md)
```

## Database

### New articles

- [.NET Aspire MongoDB database component](../database/mongodb-component.md)
- [.NET Aspire MySQL database component](../database/mysql-component.md)

> - [Title not found in: #131](../database/quickstart-sql-server.md)

Reviewer: ```suggestion
- [Tutorial: Connect an ASP.NET Core app to SQL Server using .NET Aspire and Entity Framework Core](../database/quickstart-sql-server.md)
```

## Deployment

> ### New articles
> 
> - [Title not found in: #134](../deployment/k8s-deployment.md)
> 

Reviewer: ```suggestion
```
